plugins/catalog-backend-module-bitbucket-cloud/package.json

```diff
@@ -51,6 +51,7 @@
     "@backstage/catalog-client": "workspace:^",
     "@backstage/catalog-model": "workspace:^",
     "@backstage/config": "workspace:^",
+    "@backstage/events": "workspace:^",
```

Comment:
Do we still need `plugins-events-node` here?

Reply:
same for other packages

Reply:
We do since the `SubTopicEventRouter` is still in events-node 🤔